docker-compose.yml

```diff
@@ -1,12 +1,10 @@
 version: "3.8"
+
 services:
-  client:
+  webapp:
     stdin_open: true
     build:
       context: .
-      dockerfile: Dockerfile.dev
```

Comment:
Extracted the contents from the `Dockerfile.dev` andused multi-stage build within the Dockerfile and target that stage to be built in this compose file.